pkg/apis/core/types_internalsecret.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
For consistency,
```suggestion
// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
```
and in other new files in the PR.
Or just remove the current license and run `make add-license-headers`.

Reply:
Done, PTAL :)